index.html

```diff
@@ -46,7 +46,7 @@ <h1>My Recent Works</h1>
         <li class="item">
           <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
```

Comment:
- [ ] kindly add a meaningful alt text for your placeholder images, because alts are always useful when it comes to UX and accessibilities so let's make sure that every image used has its own meaningful alt attribute